tests/test_quantity.py

```diff
@@ -146,12 +146,16 @@ def test_to_20():
     with pytest.raises(ValueError) as e_info:
         convert = v.to("radian s^-1")
 
+    assert e_info.value.args[0] == "Incompatible conversion from Hz : to radian s^-1"
```

Comment:
This kind of test would be more stable if the implementation used a new exception type (which could inherit from ValueError) with the two unit strings as construction arguments. Then you can easily construct the expected exception in the test for comparison.

Reply:
Done. Thank you.